src/main/java/com/spacesongr/songr/models/Album.java

```diff
@@ -1,18 +1,29 @@
 package com.spacesongr.songr;
```

Comment:
double check this package name - see your AlbumRepository for an example of how to do it